lib/node_modules/@stdlib/math/base/special/rcbrt/benchmark/c/cephes/benchmark.c

```diff
@@ -0,0 +1,142 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* Benchmark Cephes.
+*/
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+#include <time.h>
+#include <sys/time.h>
+
+#define NAME "rcbrt"
+#define ITERATIONS 1000000
+#define REPEATS 3
+
+#undef cbrt
```

Comment:
Comparing with [`@stdlib/math/base/special/cbrt`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/cbrt/benchmark/c/cephes/benchmark.c), is there a reason for undefining `cbrt` here? I am guessing you were getting compilation errors?

Reply:
I copied this one from the `rsqrt` folder, I can try removing it (?)

Reply:
Yeah, try removing.

Reply:
NVM. I believe it is necessary. Also in `sqrt`: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/sqrt/benchmark/c/cephes/benchmark.c#L32.